pytorch_lightning/trainer/data_loading.py

```diff
@@ -104,6 +104,7 @@ def _worker_check(self, dataloader: DataLoader, name: str) -> None:
                 )
 
         elif dataloader.num_workers <= 2 < num_cpus and not using_spawn:
+            # if changed, update the `filterwarnings` snippet in 'speed.html#num-workers'
```

Comment:
We could have a test for it, right?

Reply:
it would not guarantee that the docs gets updated if the test gets updated.